diagnostic.md

```diff
@@ -57,13 +67,13 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$('li').closest('body').hide()
 <!-- your answer ends here -->
 
 ## Question 8
 
 Use jQuery to register a click handler for clicks inside `div` items.
 
 <!-- your answer starts here -->
-
+$('div').on('click', doSomething)
```

Comment:
0.5/1. `doSomething` has to be a function that has been defined, else there needs to be a fully written out function here.